_data/sites.yml

```diff
@@ -2407,3 +2407,8 @@
   url: https://zserge.com/
   size: 20.6
   last_checked: 2022-05-13
+
+- domain: palora.vercel.app
+  url: https://palora.vercel.app/
+  size: 403
+  last_checked: 2021-7-11
```

Comment:
You probably meant to use the current year here.🙂

Reply:
but you can edit it for me!